src/components/annotate/task-detail/page-client.tsx

```diff
@@ -194,7 +196,36 @@ export function AnnotationTaskDetailPageClient({
                 disabled={lockingOutFromSpecies || lockingOutFromSex}
               />
             </div>
+            <div className="space-y-2">
+              <h3 className="text-foreground text-sm font-bold">Notes</h3>
+              <Textarea
+                className="h-[70px] resize-none text-sm overflow-y-auto"
+                placeholder="Add observations..."
+              />
+            </div>
+
+            <div className = "flex gap-3 pt-2">
+              <Button 
+                type="button"
+                variant="outline"
+                className="flex-1"
+                disabled={isFetching || isLoading}
+              >
+                <Flag className="mr-2 h-4 w-4" /> Flag Species
```

Comment:
[Suggestion]: You should replace this with "Flag Specimen", because you are flagging the whole specimen, not just the species.